.vale/fixtures/Localization/AdjectivesWithNouns/.vale.ini

```diff
@@ -0,0 +1,5 @@
+; Vale configuration file to test the `CommonlyMisusedWords` rule
```

Comment:
I'm unsure about this comment, it looks like a copy pasta

Reply:
Good catch! Updated! Thanks!